src/client/src/components/RightSidebar/index.tsx

```diff
@@ -80,9 +88,21 @@ const RightSidebar = (props: Props) => {
               />
             </div>
 
-            <div className={styles.marginLeft}>
-              <div className={styles.title}>{formatMessage(messages.yourProjectDetails)}</div>
+            {hasInvalidRequirements && (
+              <div className={styles.notificationContainer}>
+                <Notification showWarning={true} altMessage={formatMessage(messages.missingRequirements)}>
+                  {formatMessage(messages.missingRequirements)}
+                  <button
+                    className={classnames(styles.notificationLink, buttonStyles.buttonLink)}
+                    onClick={() => dispatch(ModalActions.openViewPlatformRequirementsAction())}
```

Comment:
if we decide to change name we should probably remove "View" from here too

Reply:
Done.